src/Config.js

```diff
@@ -43,6 +44,16 @@ class ConfigOptions {
         for (let v of allOptions) {
             assert(v !== undefined );
         }
+
+        let developerPairs = this._coreDevelopers.split(',').map(i => i.trim());
+        this._coreDeveloperIds = new Map();
+        for (let pair of developerPairs) {
+            let p = pair.split('=').map(i => i.trim());
+            assert(p.length === 2);
+            assert(!this._coreDeveloperIds.has(p[0]));
+            this._coreDeveloperIds.set(p[0], Number(p[1]));
```

Comment:
Please assert that the ID string parsed successfully _and_ that it is _not_ zero. The rest of the code depends on that non-falsy assumption AFAICT.